### ISSUE TITLE
Obfuscation

### ISSUE THREAD

**Issue by hot007** [AusClimateService/agcd-csiro#1]
Noting this is a public repo, for involved users the code snippets in the readme are very useful, however there may be an argument to obfuscate the filepaths and especially user name - or make the repo private to the org if possible?
Ping @DamienIrving

**Comment by DamienIrving** [comment]
Good point, @hot007. I've just removed usernames (https://github.com/AusClimateService/agcd-csiro/commit/f4786d40d8e8b79d0147526d047dd7d1bd173171) - do you think it's important to obfuscate paths too? 

(Of course the username still exists in previous commits, so I'll need to wipe the whole commit history if we leave this repo public. I'm inclined to leave it public if possible because I sometimes flick the information about the copy of AGCD on NCI to people outside of the ACS.)

**Comment by hot007** [comment]
Honestly I think it's okay, just something to be aware of going forward I guess. Given it's probably a short commit history it may be worth scrubbing though? Up to you - I'm not a security expert, I just noticed a 'bad thing' ;-) I don't think it was anything that particularly needed to be secret though.
